Sources/NetworkProtection/NetworkProtectionDeviceManager.swift

```diff
@@ -312,9 +312,11 @@ public actor NetworkProtectionDeviceManager: NetworkProtectionDeviceManagement {
     // swiftlint:enable function_parameter_count
 
     private func handle(clientError: NetworkProtectionClientError) {
+#if os(macOS)
         if case .invalidAuthToken = clientError {
             try? tokenStore.deleteToken()
         }
```

Comment:
I'm inclined to remove this, and never delete the token except through the official Privacy Pro flows - what do you think @diegoreymendez?